backend/document-service/src/main/resources/http/documentControllerRequests.http

```diff
@@ -20,6 +18,17 @@ Authorization: Bearer <access_token>
 Content-Type: application/json
 
 ### Get all document
-GET http://localhost:8083/public/api/v1/document/all?sort=type,desc
+GET http://localhost:8083/public/api/v1/document/all?sort=orderDate,desc
+Authorization: Bearer <access_token>
+Content-Type: application/json
+
+### Get document
+PUT http://localhost:8083/public/api/v1/document/638759f44da9805de8877486
 Authorization: Bearer <access_token>
-Content-Type: application/json
\ No newline at end of file
+Content-Type: application/json
+
+{
+  "orderDate": "2022-10-23",
+  "type": "WORK_STATEMENT",
+  "status": "IN_PROGRESS"
```

Comment:
I think we should pass only the `status` field here

Reply:
fixed